### ISSUE TITLE
Can you provide the complete code for training?

### ISSUE THREAD

**Issue by zzy1026** [kakaobrain/g2pM#7]
Hi, 
Thanks for the good job.
I used chinese bert to do this work with your dataset, but I could't get good result like yours. So I want to study your code and know how to training model using this datasets. But the codes your offerring now are only to predict. So can you provide the complete code about your model or bert in your papper?
Thank you for your help.

**Comment by seanie12** [comment]
Hi, in my experiment the performance of bert is really sensitive to the choice of optimizer and learning rate.
I will upload the scripts for training bert as soon as possible.

**Comment by zzy1026** [comment]
> Hi, in my experiment the performance of bert is really sensitive to the choice of optimizer and learning rate.
> I will upload the scripts for training bert as soon as possible.

Thank you!
I also want to know that in your bert, the output of fc layer is the number of this polyphonic character's possible pinyin classes or all polyphonic characters?

**Comment by seanie12** [comment]
Yes the number of output of fc layer is the number of all possible pinyins for polyphonic character.

Here is the [code](https://gist.github.com/seanie12/ba12b5d1154f5012208cda81d9f489c3) for training bert.
